scheduler/src/cook/components.clj

```diff
@@ -391,6 +396,11 @@
                                        @(lazy-load-var 'cook.spnego/require-gss)
                                        {:json-value "kerberos"}))
                                    :else (throw (ex-info "Missing authorization configuration" {}))))
+     :impersonation-middleware (fnk [[:config {authorization-config nil}]]
+                                    (letk [[{impersonators nil}] authorization-config]
```

Comment:
`(letk [[{impersonators nil}] authorization-config]`
vs
`(letk [{impersonators :impersonators, :or {impersonators nil}} authorization-config]`

Reply:
Makes sense, thanks

Reply:
I just realized that—since the default is just `nil`—this is equivalent to `(let [{impersonators :impersonators} authorization-config]`. I'm going to use that instead.